packages/astro/src/assets/fonts/vite-plugin-fonts.ts

```diff
@@ -195,41 +201,37 @@ export function fontsPlugin({ settings, sync, logger }: Options): Plugin {
 			}
 		},
 		async buildEnd() {
-			resolvedMap = null;
-
-			if (sync) {
-				hashToUrlMap = null;
-				storage = null;
+			if (sync || settings.config.experimental.fonts!.families.length === 0) {
```

Comment:
Just confirming that we can be sure `font` is always defined. Non-null assertions always make me nervous.

Reply:
Yes we're sure at this point, because we vite plugin is only created if fonts is defined